src/main/webapp/projectReview/ProjectReview.js

```diff
@@ -175,7 +181,9 @@ class ProjectReview extends Component {
   }
 
   isAdmin() {
-    return this.props.isAdmin === "true";
+    User.getUserSession(this.props.sessionUserUrl).then(resp => {
+      this.setState({isAdmin: resp.data.isAdmin});
+    });
```

Comment:
We should consider caching this at the page level. Every time `isAdmin` is called, we make a rest call, which is probably unnecessary.

Reply:
Also, consider moving admin to props. A user's admin status doesn't really change so it doesn't need to be in state. Try to use state only for things that change, and compute as much as possible.

Reply:
Greg I reverted my changes, If you agree we will refactor user role code related in US https://broadinstitute.atlassian.net/browse/BTRX-533. 
I finish with logical delete, I'll let you know as soon as walter finish with testing.